### PR TITLE
losetup: added example for --read-only and --partscan

### PR DESCRIPTION
**Pull request by z7y8hsBP**
read-only losetup and partscan are commonly used options in digital forensics. It is benefitial to know about '--partscan' as it sometimes gives other results then losetup and partprobe/kpartx, e.g. with BSD-style partitions.

]x] The page has 8 or fewer examples.
]x] The PR title conforms to the recommended templates.
]x] The page follows the content guidelines.

### CODE REVIEW

File: pages/linux/losetup.md [hunk 1]
```diff
@@ -17,3 +17,11 @@
 - Detach a given loop device:
 
 `sudo losetup -d /dev/{{loop}}`
+
+- Attach a file to the new free loop device. Scan the device for partitions:
```
Comment (zdroid) [inline pages/linux/losetup.md]:
```suggestion
- Attach a file to a new free loop device and scan the device for partitions:
```

File: pages/linux/losetup.md [hunk 2]
```diff
@@ -17,3 +17,11 @@
 - Detach a given loop device:
 
 `sudo losetup -d /dev/{{loop}}`
+
+- Attach a file to a new free loop device and scan the device for partitions:
```
Comment (agnivade) [inline pages/linux/losetup.md]: Please add these examples before the detach ones.

Comment (sbrl) [reply pages/linux/losetup.md]: @z7y8hsBP not sure I see how this is resolved, unless you opened a new PR

Comment (z7y8hsBP) [reply pages/linux/losetup.md]: Hmm, I added and commited it in my fork, should automatically add it to the PR. But I can open a new PR.

Comment (z7y8hsBP) [reply pages/linux/losetup.md]: https://github.com/z7y8hsBP/tldr/commit/c8ac8c6f88e552b7e342fe167d8b6ce61ab32d3c#diff-39cc51be6e08db5cd9c9816ce9003387

Comment (sbrl) [reply pages/linux/losetup.md]: Ah, I see @z7y8hsBP. Perhaps it's because this PR has already been merged. Thanks for opening a new one :-)

**Pull request merged**